bin/commands/init/generate/index.ts

```diff
@@ -0,0 +1,59 @@
+/*
+// This program and the accompanying materials are made available
+// under the terms of the Eclipse Public License v2.0 which
+// accompanies this distribution, and is available at
+// https://www.eclipse.org/legal/epl-v20.html
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as os from "cm_os";
+import * as xplatform from "xplatform";
+import * as fs from '../../../libs/fs';
+import * as config from '../../../libs/config';
+import * as common from '../../../libs/common';
+import * as zosFs from '../../../libs/zos-fs';
+import * as zosJes from '../../../libs/zos-jes';
+
+export function execute(dryRun?: boolean) {
+  common.requireZoweYaml();
+  const ZOWE_CONFIG=config.getZoweConfig();
+  const tempFile = fs.createTmpFile();
+  zosFs.copyMvsToUss(ZOWE_CONFIG.zowe.setup.dataset.prefix + '.SZWESAMP(ZWEGENER)', tempFile);
+  let jclContents = xplatform.loadFileUTF8(tempFile, xplatform.AUTO_DETECT);
+
+  jclContents = jclContents.replace(/\{zowe\.setup\.dataset\.prefix\}/gi, ZOWE_CONFIG.zowe.setup.dataset.prefix);
+  jclContents = jclContents.replace(/\{zowe\.runtimeDirectory\}/gi, ZOWE_CONFIG.zowe.runtimeDirectory);
+  let absConfig = fs.convertToAbsolutePath(std.getenv('ZWE_PRIVATE_CONFIG_ORIG'));
```

Comment:
This works only for unix file.
Any combination of `FILE(zowe.yaml)` and `PARMLIB(ZOWE(ZWEYAML))` will fail.
In other words, `zwe init generate -c unix_file_only`?

Reply:
thanks, i have an idea for how to fix this.
if FILE or PARMLIB seen, parse to do absolute conversion on each FILE entry seen.